src/Main.java

```diff
@@ -1,9 +1,43 @@
+import java.util.HashMap;
+import java.util.Scanner;
 
 public class Main {
     public static void main(String[] args) {
 
-        // Hier mag je je code schrijven voor de hoofd-opdracht
+        int[] numeric = {1, 2, 3, 4, 5, 6, 7, 8, 9, 0};
 
+        String[] alphabetic = {"een", "twee", "drie", "vier", "vijf", "zes", "zeven", "acht", "negen", "nul"};
 
+        Translator translator = new Translator(numeric, alphabetic);        // Hier mag je je code schrijven voor de hoofd-opdracht
+
+        boolean play = true;
+        String ongeldig = "ongeldige invoer";
+        Scanner scanner = new Scanner(System.in);
+
+
+        while (play) {
+            // Print opties
+            System.out.println("Type 'x' om te stoppen");
+            System.out.println("Type 'v' om te vertalen");
+
+            // Lees de input van de gebruiker
+            String input = scanner.nextLine();
+
+            if (input.equals("x")) {
+                play = false;
+            } else if (input.equals("v")) {
+                System.out.println("Type een cijfer in van 0 t/m 9");
+                int number = scanner.nextInt();
+                scanner.nextLine(); // voeg deze toe om de newline te consumeren
+                if (number < 10) {
```

Comment:
(number < 10 && number >= 0)...in de uitwerkingen van Novi gezien...idd...alleen wie zou min drie (-3) intypen :)...als je -3 intypt krijg je "De vertaling van -3 is nul" dus idd fout (bij mij ook idd)